src/css/NavLinks.css

```diff
@@ -0,0 +1,20 @@
+/* Nav-Link*/
+.navlink {
+  color: #f4a431;
+  text-decoration: none;
+}
+
+.navlink-active {
+  font-weight: 900;
```

Comment:
Do we have a font weight of 900? If we're using our `font-family: 'Source Sans Pro', sans-serif;` as our default font in our `App.css`, I think we only have 300 and 400 weights downloaded for that font. Please correct me if I'm wrong because I don't know a ton about font weights.